source/core_http_client.c

```diff
@@ -1186,6 +1254,8 @@ static HTTPStatus_t addHeader( HTTPRequestHeaders_t * pRequestHeaders,
     char * pBufferCur = NULL;
     size_t toAddLen = 0u;
     size_t backtrackHeaderLen = pRequestHeaders->headersLen;
+    const uint8_t isField = 1;
+    const uint8_t isValue = 0;
```

Comment:
This private function is called throughout many public-facing functions, so I'm wondering if we could avoid the extra stack usage by having the above as an enum or macro.

Reply:
sure